app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,36 @@
+class BookingsController < ApplicationController
+  before_action :set_booking, only: [:show]
+  def index
+    @bookings = policy_scope(Booking).order(created_at: :desc)
+  end
+
+  def new
+    @booking = Booking.new
```

Comment:
same comment, you will probably need to reference the @space in the view with the form so you should define it
```suggestion
    @space = Space.find(params[:space_id])
    @booking = Booking.new
```